pkg/render/manager.go

```diff
@@ -49,11 +49,11 @@ import (
 )
 
 const (
-	managerPort           = 9443
+	ManagerPort           = 9443
 	managerTargetPort     = 9443
 	ManagerServiceName    = "tigera-manager"
 	ManagerDeploymentName = "tigera-manager"
```

Comment:
I'm wondering if we want to change the prefix of all resources to "calico-". We've been doing that for other components and it would be nice over time to end up in a fully consistent state.